carelink.js

```diff
@@ -2,13 +2,16 @@
 "use strict";
 
 var _ = require('lodash'),
-    common = require('common'),
-    qs = require('qs'),
+    axios = require('axios').default,
+    axiosCookieJarSupport = require('axios-cookiejar-support').default,
+    tough = require('tough-cookie'),
     urllib = require('url'),
-    request = require('request');
+    qs = require('qs');
 
 var logger = require('./logger');
 
+process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
```

Comment:
Nervous about shipping this to production.  We would like to validate that we are talking to Medtronic without someone tampering.

Reply:
the current code not using this